health/notifications/email/metadata.yaml

```
@@ -0,0 +1,70 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-email'
+  meta:
+    name: 'Email'
+    link: ''
+    categories:
+      - notify.agent
+    icon_filename: 'email.png'
+  keywords:
+    - email
+  overview:
+    notification_description: |
+      Send notifications via Email using Netdata's Agent alert notification feature, which supports dozens of endpoints, user roles, and more.
+    notification_limitations: ''
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - A working sendmail command for email alerts to work. Almost all MTAs provide a sendmail interface. Netdata sends all emails as user netdata, so make sure your sendmail works for local users.
+            - Access to the terminal where Netdata Agent is running
+    configuration:
+      file:
+        name: 'health_alarm_notify.conf'
+      options:
+        description: 'The following options can be defined for this notification'
+        folding:
+          title: 'Config Options'
+          enabled: true
+        list:
+          - name: 'EMAIL_SENDER'
+            default_value: 'netdata'
+            description: "You can change `EMAIL_SENDER` to the email address sending the notifications."
+            required: false
+          - name: 'SEND_EMAIL'
+            default_value: ''
```

```suggestion
            default_value: 'YES'
```

I think the default is YES.